examples/ScanLoop.cpp

```diff
@@ -0,0 +1,65 @@
+//////////////////////////////////////////////////////////////////////////////////////
+// Copyright 2020 Lawrence Livermore National Security, LLC and other CARE developers.
+// See the top-level LICENSE file for details.
+//
+// SPDX-License-Identifier: BSD-3-Clause
+//////////////////////////////////////////////////////////////////////////////////////
+
+// This macro needs to be defined before including care/care.h,
+// which allows you to port a file at a time. Without this define,
+// LOOP_STREAM will run on the CPU. With this define, LOOP_STREAM
+// will run on the GPU only if you built with CUDA enabled. Otherwise,
+// if CUDA is disabled, it will default back to running on the host.
+#define GPU_ACTIVE
+
+// CARE library header
+#include "care/care.h"
+#include "care/scan.h"
+
+// Std library headers
+#include <iostream>
+#include <vector>
+
+int main(int, char**) {
+   // Normal code
+   int size = 1000;
+   std::vector<int> data;
+
+   int threshold = 420;
+
+   for (int i = 0; i < size; ++i) {
+      int value = i * i;
+
+      if (value < threshold) { 
+         data.push_back(value);
+      }
+   }
+
+   std::cout << "CPU count: " << data.size() << std::endl;
+
+   // Ported code
+   int size2 = 1000;
+   care::host_device_ptr<int> data2(size2);
```

Comment:
Comment on why you are creating a host_device pointer, which can has data on both the GPU and CPU. Without this, the SCAN_LOOP would be invalid, you can't use a regular c array etc etc

Reply:
Done.